src/Symfony/Bridge/Monolog/Logger.php

```diff
@@ -20,52 +19,10 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class Logger extends BaseLogger implements LoggerInterface, DebugLoggerInterface
+class Logger extends BaseLogger implements DebugLoggerInterface
 {
     /**
-     * @see Symfony\Component\HttpKernel\Log\DebugLoggerInterface
-     * @deprecated since version 2.2, to be removed in 3.0. Use emergency() which is PSR-3 compatible.
-     */
-    public function emerg($message, array $context = array())
-    {
-        trigger_error('The '.__METHOD__.' method inherited from the Symfony\Component\HttpKernel\Log\LoggerInterface interface is deprecated since version 2.2 and will be removed in 3.0. Use the emergency() method instead, which is PSR-3 compatible.', E_USER_DEPRECATED);
-
-        return parent::addRecord(BaseLogger::EMERGENCY, $message, $context);
-    }
-
-    /**
-     * @deprecated since version 2.2, to be removed in 3.0. Use critical() which is PSR-3 compatible.
-     */
-    public function crit($message, array $context = array())
-    {
-        trigger_error('The '.__METHOD__.' method inherited from the Symfony\Component\HttpKernel\Log\LoggerInterface interface is deprecated since version 2.2 and will be removed in 3.0. Use the method critical() method instead, which is PSR-3 compatible.', E_USER_DEPRECATED);
-
-        return parent::addRecord(BaseLogger::CRITICAL, $message, $context);
-    }
-
-    /**
-     * @deprecated since version 2.2, to be removed in 3.0. Use error() which is PSR-3 compatible.
-     */
-    public function err($message, array $context = array())
-    {
-        trigger_error('The '.__METHOD__.' method inherited from the Symfony\Component\HttpKernel\Log\LoggerInterface interface is deprecated since version 2.2 and will be removed in 3.0. Use the error() method instead, which is PSR-3 compatible.', E_USER_DEPRECATED);
-
-        return parent::addRecord(BaseLogger::ERROR, $message, $context);
-    }
-
-    /**
-     * @deprecated since version 2.2, to be removed in 3.0. Use warning() which is PSR-3 compatible.
-     */
-    public function warn($message, array $context = array())
-    {
-        trigger_error('The '.__METHOD__.' method inherited from the Symfony\Component\HttpKernel\Log\LoggerInterface interface is deprecated since version 2.2 and will be removed in 3.0. Use the warning() method instead, which is PSR-3 compatible.', E_USER_DEPRECATED);
-
-        return parent::addRecord(BaseLogger::WARNING, $message, $context);
-    }
-
-    /**
      * {@inheritdoc}
->>>>>>> 2.7
```

Comment:
From the PR?

This is also present in https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Monolog/Logger.php#L68